### DISCUSSION TITLE
ant-design-vue-pro：登录Login请求中then不执行

### DISCUSSION THREAD

**Discussion by qadan72** [ant-design/ant-design#46382] [Q&A] [answered]
**Question （问题描述）**
我的操作如下：
1，git clone，yarn install
2，注释mock
3，使用proxy更改后端地址（配置方法正确）
4，在登录页面，点击登录后，network有请求地址，且请求成功200，但是then方法不执行，then方法有个console.log不执行，catch执行了，error的内容就是network响应的内容，这是为什么

**Describe the solution you'd like （你期待的是什么？）**
请问是哪里的问题？
`Login(loginParams)
            .then((res) => {
              console.log(6666666666)
              this.loginSuccess(res)
            })
            .catch(err => this.requestFailed(err))
            .finally(() => {
              state.loginBtn = false
            })`

**Additional context（附加信息）**

![144204vh0aq2ih2520kt5a](https://github.com/vueComponent/ant-design-vue-pro/assets/40349902/187c23ca-159e-4ada-8f78-b277e696cbf5)

**Comment by li-jia-nan** [comment]
来错地方了，哥，这里是 antd

**Reply by li-jia-nan** [reply]
你要找的应该是：https://github.com/vueComponent/ant-design-vue-pro

**Reply by qadan72** [reply]
我知道，这里人多，所以来这个问问，提了issues没人回答;https://github.com/vueComponent/ant-design-vue-pro/issues/1445

**Reply by li-jia-nan** [reply]
不是一个团队，你来这里也没用……

**Comment by qadan72** [comment, marked as answer]
问题解决了，login.vue登录接口调用的vuex方法，这里有拦截器，注释掉就行了，给后面的人参考下
